cachito/workers/config.py

```diff
@@ -31,7 +31,7 @@ class Config(object):
     cachito_default_environment_variables = {
         "gomod": {
             "GOSUMDB": {"value": "off", "kind": "literal"},
-            "GOTOOLCHAIN": {"value": "local", "kind": "literal"},
+            "GOTOOLCHAIN": {"value": "auto", "kind": "literal"},
```

Comment:
Did we want to override the user-side GOTOOLCHAIN with auto?

Reply:
Is this user-side? I thought this is for cachito itself, i.e. the default that will be used for the prefetch.

Reply:
Yes, these should get passed through to the request output: https://github.com/containerbuildsystem/cachito/blob/59967a1341ad4ee99c43f5492459c53e6e781180/cachito/workers/tasks/gomod.py#L171